src/components/views/rooms/MessageComposerButtons.tsx

```diff
@@ -339,7 +339,7 @@ class PollButton extends React.PureComponent<IPollButtonProps> {
 
     public render() {
         // do not allow sending polls within threads at this time
-        if (this.props.relation?.rel_type === RelationType.Thread) return null;
+        if (this.props.relation?.rel_type === THREAD_RELATION_TYPE.name) return null;
```

Comment:
Shouldn't this use `THREAD_RELATION_TYPE.matches(relation?.rel_type)` to support both legacy and new?